onnxruntime/core/platform/posix/env.cc

```diff
@@ -228,7 +228,8 @@ class PosixThread : public EnvThread {
         auto ret = pthread_setaffinity_np(pthread_self(), sizeof(cpu_set_t), &cpuset);
         if (ret != 0) {
           auto [err_no, err_msg] = GetSystemError(ret);
-          LOGS_DEFAULT(ERROR) << "pthread_setaffinity_np failed for thread: " << pthread_self()
+          LOGS_DEFAULT(ERROR) << "pthread_setaffinity_np failed for thread: " << syscall(SYS_gettid)
```

Comment:
Will this work for Mac OS? https://stackoverflow.com/questions/58643784/syscallsys-gettid-return-1

Reply:
Not sure, this is a Linux only code. `pthread_setaffinity_np` does not work on MacOs.